src/Symfony/Component/Serializer/Mapping/Loader/AnnotationLoader.php

```diff
@@ -100,7 +100,7 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata)
                 continue;
             }
 
-            $getAccessor = preg_match('/^(get|)(.+)$/i', $method->name);
+            $getAccessor = preg_match('/^(get)(.+)$/i', $method->name);
             if ($getAccessor && 0 !== $method->getNumberOfRequiredParameters()) {
```

Comment:
let's use this instead? (no need for the `$getAccessor` variable)
`if (0 === stripos($method->name, 'get') && 0 !== $method->getNumberOfRequiredParameters()) {`

Reply:
Done in [2c752fd](https://github.com/symfony/symfony/pull/47255/commits/2c752fd97f4fda7c89c039fc4643578c9caf4a46)